report.php

```diff
@@ -91,7 +94,11 @@
 
         if (!empty($eventids)) {
             $mover = new \logstore_xapi\log\moveback($eventids);
-            $mover->execute();  //LMS-1692 set notification
+            if ($mover->execute()) {
```

Comment:
If there is an issue moving the records, do they all fail or can there be a mix between success/fail?

Reply:
I don't think there is any case when the mover will fail.

Only if the database handler throw error will the process failing. In this case we don't follow how many record were moved and how many were not.

Reply:
ok - I just wondered if we knew a count we could display

Reply:
We know count but I think we should avoid to show it :)

Reply:
If we show the numbers that will give us more discussion like as:
- "If I set 10 element to replay and I confirm to update it why only 9 was move back?"
- "Because during our system waited for your confirmation, on the background the emit_task was run and resolved one of those."

Reply:
Yeah I'm fine with that approach Lazslo.
I was just wondering if it was possible that some could fail but not others, and whether we had to consider that situation.
I'm happy with this, so there is nothing to do here.